modules/core/src/main/java/com/illposed/osc/LibraryInfo.java

```diff
@@ -286,6 +288,19 @@ public static boolean isAndroidApi23AndLower() {
 		}
 	}
 
+	/**
+	 * Checks if java.awt.color is available and the library is used on desktops
+	 */
+	public static boolean isDesktop() {
+		try {
+			Class.forName("java.awt.color");
+			return true;
+		} catch (ClassNotFoundException ignore) {
+			//my class isn't there!
```

Comment:
remove or change to something like "This class is part of JDK, so we are not on JDK -> assuming we are on Android"?

Reply:
You probably meant the StandardProtocolFamily part? I did some renaming and changed the description. I hope you like it better.

Reply:
ah.. I guess that is also a good change, though I would have been fine with it before too. ;-)
.. I was concerned about the `isDesktop()` method. Many developpers nowadays only know Android or only know the JDK world, and the method should be named in a way, and documented in a way that makes it understandable for both of these.

I say this, because I know very little about Android, and only since a year or two, know that is does not have `java.awt.Color`, for example. ;-)
.. and I like to make code save for people as ignorant as I am (was) myself.

I would probably revert the boolean value of the function, and call it `isAndroid()` instead, because JDK can be used on headless server, on embedded systems and desktops.

Reply:
I made the function's naming more generic and put the Android stuff into its descriptions.

Reply:
uff, much better then what I though of, great! :-)